applications/ConvectionDiffusionApplication/python_scripts/coupled_structural_thermal_solver.py

```diff
@@ -67,6 +67,8 @@ def __init__(self, model, custom_settings):
 
         from KratosMultiphysics.ConvectionDiffusionApplication import python_solvers_wrapper_convection_diffusion
         self.thermal_solver = python_solvers_wrapper_convection_diffusion.CreateSolverByParameters(self.model,self.settings["thermal_solver_settings"],"OpenMP")
+        solver_type = self.settings["structural_solver_settings"]["solver_type"].GetString()
+        self.is_dynamic = solver_type == "Dynamic" or solver_type == "dynamic"
```

Comment:
```suggestion
        self.is_dynamic = solver_type == "dynamic"
```
I wouldn't support old input syntax in new solvers.

Reply:
can you pleas change this?